lib/include/nitro/gx/g3b.h

```diff
@@ -0,0 +1,54 @@
+#ifndef _NITRO_GX_G3B_H_
+#define _NITRO_GX_G3B_H_
+
+#include <nitro/gx/gxcommon.h>
+#include <nitro/gx/g3.h>
+#include <nitro/gx/g3imm.h>
+
+void G3B_Nop(GXDLInfo * info);
```

Comment:
hesder guard , pointer style, prefix, also copied and pasted, please dont do this shit